CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ This is the changelog for SpotBugs. This follows [Keep a Changelog v1.0.0](http:
 Currently the versioning policy of this project follows [Semantic Versioning v2.0.0](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased - 2022-??-??
+### Changed
+- Rewrite some member in `ResourceValueFrame.java` to Enum([#2061](https://github.com/spotbugs/spotbugs/issues/2061))
```

Comment:
This is a major, breaking change, IMO it would be better, if the changelog contained that it's a breaking change.

Reply:
Is it though?  Is there external usage of this?  I don't see it in the gradle or maven plugins, maybe something else is using it.  Can call it breaking I guess given its public got thinking its ok.